backend/users/Users.ts

```diff
@@ -9,24 +10,43 @@ export async function findOrCreateLocalUser(email) {
 
   let localUser = users.length > 0 ? users[0] : null;
 
+  const hashLocalDevPass = await bcrypt.hash("localDevPassword", 5);
+
   if (!localUser) {
     localUser = await sequelize.models.User.create({
       firstName: "localDevFirstName",
       lastName: "localDevLastName",
       email: email,
-      password: "localDevPassword",
+      password: hashLocalDevPass,
       googleAuthToken: null,
     });
   }
+
   return localUser;
 }
 
 export async function findUser(email, password) {
+  const hashpass = await bcrypt.hash(password, 5);
+
+  // Issue 1: users should be typed as an array of User objects, but was typed as an array of Models...
+  // Because of that, was not able to use user.password on Line 47
   const users = await sequelize.models.User.findAll({
     where: {
       email: email,
-      password: password,
     },
   });
-  return users.length > 0 ? users[0] : null;
+
+  // Issue 2: I couldn't use users.first so went with the below line
```

Comment:
Why couldn't you use `users.first`? Typescript? If you ts-ignore it, does it work?

Reply:
.first() was incorrect syntax for Sequelize.